src/helm/benchmark/vlm_run_specs.py

```diff
@@ -39,6 +39,37 @@ def get_vlm_generation_adapter_spec(
     )
 
 
+def get_short_answer_generation_adapter_spec():
+    return get_generation_adapter_spec(
+        instructions="Just give a short answer without answering in a complete sentence.",
+        max_tokens=20,
+    )
+
+
+def get_multiple_choice_joint_adapter_spec(
+    input_noun: Optional[str],
+    output_noun: str,
+    instructions: str = "",
+    max_train_instances: int = 0,
+) -> AdapterSpec:
+    return AdapterSpec(
+        method=ADAPT_MULTIPLE_CHOICE_JOINT_MULTIMODAL,
+        global_prefix="",
+        instructions=instructions,
+        input_prefix=f"{input_noun}: " if input_noun is not None else "",
+        input_suffix="\n" if input_noun is not None else "",
+        output_prefix=f"{output_noun}: ",
+        output_suffix="\n",
+        instance_prefix="\n",
+        max_train_instances=max_train_instances,
+        num_outputs=1,
+        max_tokens=2,
```

Comment:
why two tokens here? it should be just a letter no? Is this due to some spaces being generated?

Reply:
I set the max to 2; otherwise, Gemini throws an error.